tests/gdb_lots_of_threads_test.c

```diff
@@ -160,6 +166,10 @@ int main(int argc, char* argv[])
       }
    }
 
+   while (wait_for_gdb != 0) {
+      usleep(100);
+   }
```

Comment:
I assume the intention here is for the user of gdb to set wait_for_gdb to zero once gdb is attached.
Maybe add a message telling the user to set wait_for_gdb = 0 to let the test proceed.

Reply:
Yes. Actually there is "-w" option to set `wait_for_gdb` to 0, and we use in in bats.